### ISSUE TITLE
10% lab marks

### ISSUE THREAD

**Issue by SweetPotato0213** [nus-cs2030/2021-s2#326]
Is it true that we just need to complete 5 out of the 8 labs in order to get the 10% marks?

**Comment by elizabethh07** [comment]
it's 5% but sorry i'm not sure how they grade the labs

**Comment by marvinljw** [comment]
yes thats what i heard too

**Comment by Onelayer** [comment]
Yes, based on luminus:
Weekly labs (5%) -- 5 out of 7 correct labs

**Comment by Zekeseoj** [comment]
but we have 8 labs so which one is not counted?

**Comment by Brandon-Chiu** [comment]
I believe they will take the best 5 submissions.

**Comment by Brandon-Chiu** [comment]
I think they will take the best 5 submissions

**Comment by darrylcjy** [comment]
Yea, as long as you have 5 correct lab submissions (A grade), you should be fine!

**Comment by Jason-Ng00** [comment]
Yup having 5 out of 7 completed lab (getting A on codecrunch) should be fine, but the labs are a good practice for the lecture content so try to attempt all of them if you have time.

**Comment by JoelToh404** [comment]
I believe that getting A on codecrunch means that the lab is completed.

**Comment by Martini-Chwang** [comment]
Seems like getting 5x As (100/100) would be sufficient.

**Comment by Jaryl-Ong** [comment]
Yup as long as you got an A for 5 of them

**Comment by kekekexinnn** [comment]
Taking best 5 (A) out of 8 for labs

**Comment by ashwinkc-29** [comment]
5 A grades from your labs on codecrunch is sufficient.

**Comment by yowyuxuan** [comment]
u need a total of 5As out of 8 labs, doesn't matter if you didn't complete all 8 labs.

**Comment by Shijia003** [comment]
yup, but make sure your 5 labs are all full marks

**Comment by W-ShiHan** [comment]
Each lab completion gives you 1%, capped at 5%

**Comment by JWulaXia** [comment]
yes, 5/8
